gnomad_constraint/constraint/constraint_pipeline.py

```diff
@@ -6,14 +6,22 @@
 import hail as hl
 
```

Comment:
I think it make sense to leave this line blank. `hail` usually was in the chunk that import pandas and other package in gnomad_method. It can still in this chunk even if it's the only one imported in this case.